ec2utils/ec2deprecateimg/lib/ec2utils/ec2deprecateimg.py

```diff
@@ -120,27 +119,29 @@ def _get_all_type_match_images(self, filter_replacement_image=None):
         """Get all images that match thespecified virtualization type.
            All images owned by the account if not type is specified."""
         images = []
-        my_images = self.ec2.get_all_images(owners='self')
+        my_images = self.ec2.describe_images(Owners=['self'])['Images']
```

Comment:
Ooh. Did you want to use Client#describe_images, and work with the dicts, or did you want to stick with objects & attributes, using the resource?